WebApplication/ClientApp/src/components/forgePdfViewExtension.js

```diff
@@ -20,6 +20,10 @@ const Autodesk = window.Autodesk;
 
 function ForgePdfViewExtension(viewer, options) {
     Autodesk.Viewing.Extension.call(this, viewer, options);
+
+    this.subToolbar = null;
+    this.prevbutton = new Autodesk.Viewing.UI.Button('drawing-button-prev');
+    this.nextbutton = new Autodesk.Viewing.UI.Button('drawing-button-next');
```

Comment:
Please use [camelCase](https://github.com/airbnb/javascript#naming--camelCase) for var names.